libcodechecker/libhandlers/analyze.py

```diff
@@ -130,6 +130,16 @@ def add_arguments_to_parser(parser):
                         help="Specify the format the analysis results should "
                              "use.")
 
+    parser.add_argument('-c', '--clean',
+                        dest="clean",
+                        required=False,
+                        action='store_true',
+                        default=False,
```

Comment:
The default is explain the help. We use `default=argparse.SUPPRESS` so the help formatter doesn't create a `(default: False)` to the help output.

**NOTE!** If default is `suppress`, you can't use `if args.clean:` as it will be an error if `--clean` is not given, you need to use `if 'clean' in args` instead!